samples/client/petstore/cpp-qt5/client/PFXStoreApi.cpp

```diff
@@ -127,6 +145,10 @@ void PFXStoreApi::getInventory() {
                            .arg(_basePath)
                            .arg("/store/inventory");
 
+    addHeaders("api_key",_apiKeys.find("api_key").value());
```

Comment:
If empty, it would be better to not inject this in the header.

Reply:
Yes. I will change that. Thanks!